raiden/ui/cli.py

```diff
@@ -157,8 +158,8 @@ def options(func):
                 'The "production" setting adds some safety measures and is mainly intended '
                 'for running Raiden on the mainnet.\n'
             ),
-            type=click.Choice([e.value for e in Environment]),
-            default=Environment.DEVELOPMENT.value,
+            type=EnvironmentChoiceType([e.value for e in Environment]),
+            default=Environment.PRODUCTION.value,
```

Comment:
If you're using `EnvironmentChoiceType`, the return of it is `Environment`. Mustn't `default` be of type `Environment` instead of `str`, i.e. `default=Environment.PRODUCTION`?

Reply:
@andrevmatos not sure. This currently works as it is. So I assume that the default gets fed to the `EnivronmentChoiceType`. @ulope any hints?

Reply:
Yes, the default is passed through the conversion function. So should be fine as is.